plugins/@grouparoo/pipedrive/jest.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  testEnvironment: "node",
+  testTimeout: 240000,
```

Comment:
This is huge! Do you need to wait that long because the API is slow?  Most plugins look like this:


```
const { helper } = require("@grouparoo/spec-helper");

module.exports = {
  testEnvironment: "node",
  testTimeout: helper.defaultTime,
  setupFiles: ["<rootDir>/jest.setup.js"],
  transform: {
    "^.+\\.ts?$": "ts-jest",
  },
  testPathIgnorePatterns: ["<rootDir>/src", "<rootDir>/dist"],
};
```

Reply:
Nope! I must have copied this from somewhere. Fixed